styles/style.css

```diff
@@ -1,3 +1,154 @@
+@import 'normalize.css';
+@import url('https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap');
+@import url('https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css');
+
+:root {
+  --primary: deepPink;
+  --headerBg: #1d1d26;
+  --textColor: #0e0f0f;
+  --textColorInverse: #fcfdff;
+  --bg: #fcfdF0;
+  --bgTint: #dfeded;
+  --bgGrey: #e6e8e8;
+  --white: #fcfdff;
+  --linkColor: deepPink;
+  --border-color: #0e0f0f;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --bg: #161618;
+    --bgTint: #27272c;
+    --textColor: #dbd7db;
+    --bgGrey: #27272c;
+    --textColorInverse: #0e0f0f;
+    --linkColor: #70c989;
+    --border-color: var(--white);
+  }
+}
+
+* {
+  color: var(--textColor);
+}
+
+a {
+  color: var(--linkColor);
+}
+
 body {
-  background-color: tomato;
+  font: 16px Helvetica, Sans-Serif;
+  line-height: 24px;
+  background-color: var(--bg);
+}
+
+p {
+  line-height: 18px;
+}
+
+h1 {
+  margin: 0 0 16px 0;
+  padding: 0 0 16px 0;
+  font-size: 42px;
+  font-weight: bold;
+  letter-spacing: -2px;
+}
+
+.resume__section_content__list__item span {
+  font-style: italic;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 16px;
+  color: #999;
+  font-weight: normal;
+  padding-left: 15px;
+  float: right;
+}
+
+.resume__section_content__list {
+  list-style-type: none;
+}
+
+.resume__section_content__list li {
+  margin-top: 10px;
+}
+
+.resume__section_content__list li:first-child {
+  margin-top: 0px;
+}
+
+body.resume {
+  width: 800px;
+  margin: 48px auto 60px;
+}
+
+.resume header {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 10px;
+}
+
+.resume__headshot {
+  margin-left: auto;
+  transform: rotate(-15deg);
+  padding-left: 10px;
+}
+.resume__header__text {
+  padding-right: 25px;
+} 
+
+
+
+ 
```

Comment:
Nit: Do we need to have this extra space here?

Reply:
Absolutely not.  Will remove